post.md

```diff
@@ -1,22 +1,22 @@
 For a while I've wanted to explore Suspense.
-I read a react blog post way back in 2019 about
+I read a React blog post way back in 2019 about
 [building great UX with React Suspense](https://legacy.reactjs.org/blog/2019/11/06/building-great-user-experiences-with-concurrent-mode-and-suspense.html).
-I remember it giving me a real aha moment.
-The description of the waterfall approach to loading data vs. loading data as you render really resonated with performance problems we'd had in a project I was working with at the time.
-Suspense though was experimental in react 16; the only thing that it was supported with was `Lazy` components,
-and I never like using experimental API's in production
+I remember it giving me a real "aha!" moment.
+The description of the waterfall approach to loading data vs loading data as you render really resonated due to performance problems we'd had in a project I was working with at the time.
+Suspense though was experimental in React 16; the only thing that it was supported with was `Lazy` components,
+and I never like using experimental APIs in production
 so I eagerly awaited the arrival of a stable API.
 
 But that day never arrived. I was put off and lost interest.
```

Comment:
Do I really have to go off and read that side piece before I can continue with this post? I need my own "hook" to draw me in and so far I don't know why you are excited, don't really have a gist of the performance problem and know that you lost interest. Draw me in with something more juicy please!